functests/performance/updating_profile.go

```diff
@@ -113,12 +118,55 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 			}
 		}
 	})
+
+	It("[test_id:28440]Verifies that nodeSelector can be updated in performance profile", func() {
+		var newWorkerNode corev1.Node
+		newRole := "worker-test"
+		newLabel := fmt.Sprintf("%s/%s", testutils.LabelRole, newRole)
+		newNodeSelector := map[string]string{newLabel: ""}
+
+		By("Skipping test if cluster does not have another available worker node")
+		workerNodes, err := nodes.GetByRole(testclient.Client, "worker")
+		Expect(err).ToNot(HaveOccurred())
+
+		for i := range workerNodes {
```

Comment:
hm, since you only read from the slice, it's safe to use the value from the range, and not the index (`_, node := ...`). It's a bit nicer.

Reply:
right, thanks!